src/components/radio/radio.spec.ts

```diff
@@ -393,6 +409,40 @@ describe('MdRadio', () => {
       expect(weatherRadioInstances[1].checked).toBe(false);
       expect(weatherRadioInstances[2].checked).toBe(true);
     });
+
+    it('should add aria-label attribute to the underlying input element if defined', () => {
+      expect(fruitRadioNativeInputs[0].getAttribute('aria-label')).toBe('Banana');
+    });
+
+    it('should not add aria-label attribute if not defined', () => {
+      expect(fruitRadioNativeInputs[1].getAttribute('aria-label')).toBe(null);
```

Comment:
I'd use `hasAttribute` here and in other tests where you're checking for the _presence_ of the attribute.